config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-presets.yaml

```diff
@@ -7,3 +7,12 @@ presets:
       secretKeyRef:
         name: akeyless-test-cred
         key: credentials
+
+- labels:
+    preset-gcp-secrets-store-creds: "true"
+  env:
+  - name: GCP_SA_JSON
+    valueFrom:
+      secretKeyRef:
+        name: gcp-secrets-store-cred
```

Comment:
How is this secret going to be created in the eks prow cluster?

Reply:
This is the environment variable we need for GCP auth. For this there should be a k8s secret called gcp-secrets-store-cred available before in the cluster. https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/1588/ in the k8s creates the secret

Reply:
@BenTheElder @ameukam Do we have a way to pre-create a Kubernetes secret with creds in eks prow cluster that can be used for tests?

Reply:
We probably don't want to use k8s secrets but switch to GKE Workload identity ? There is no obligation to run this on EKS. 
_There is no indication that EKS is mandatory for E2E tests focus on GCP_

Reply:
commented https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/1588#issuecomment-2261261023

we generally run things same-cloud when possible (in the future it would be good to see this for azure as well) to minimize cross cloud traffic etc.

however, we aren't allowing the creation of arbitrary secrets in any clusters, only secrets that reference other community controlled accounts and resources, to avoid digging that hole again.

what does this secret contain? can we please discuss the GCP infra requirements in https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues/1583 ?

also, again, any non migrated jobs are going to be shut down in the coming days / week to prepare to migrate the CI control plane (See the dev@kubernetes.io announcements, chairs and techleads meetings, etc.)